x-pack/plugins/security_solution/public/app/app.tsx

```diff
@@ -17,18 +18,24 @@ import type { AppLeaveHandler, AppMountParameters } from '@kbn/core/public';
 
 import { EuiThemeProvider } from '@kbn/kibana-react-plugin/common';
 import { CellActionsProvider } from '@kbn/cell-actions';
+
+import { getComments } from '../assistant/get_comments';
+import { augmentMessageCodeBlocks, LOCAL_STORAGE_KEY } from '../assistant/helpers';
+import { useConversationStore } from '../assistant/use_conversation_store';
```

Comment:
I'd prefer to use package export for this. Is there any reason why we are not able to export it with package?

Reply:
Yeah, so this `useConversationStore` is a 'solution specific' hook for managing conversation state outside of the assistant. So it is separate from the `useConversation` hook part of the assistant package api. We can clarify this in the documentation (and hopefully this is only temporary while we use localstorage).